proxy/http/HttpProxyServerMain.cc

```diff
@@ -48,6 +49,15 @@ HttpSessionAccept *plugin_http_transparent_accept = nullptr;
 static SLL<SSLNextProtocolAccept> ssl_plugin_acceptors;
 static Ptr<ProxyMutex> ssl_plugin_mutex;
 
+// used to check whether we are ready to start the accept threads
+int wait_for_et_net_ready = 0;
```

Comment:
Why not make this std::atomic?

Reply:
ink_atomic_cas() is already atomic so I didn't make `wait_for_et_net_ready` atomic.

Reply:
We are moving towards using std::atomic instead of relying no ink_atomic_*

Reply:
Ah, then I'll make the change then.